### PR TITLE
Remove DLNA-specific DeviceProfile code

### PR DESCRIPTION
**Pull request by barronpm**
These will be moved to the plugin

### CODE REVIEW

File: MediaBrowser.Model/Dlna/DeviceProfile.cs [hunk 1]
```diff
@@ -31,104 +26,6 @@ public class DeviceProfile
         [XmlIgnore]
```
Comment (nielsvanvelzen) [inline MediaBrowser.Model/Dlna/DeviceProfile.cs:26]: Are those XmlIgnore attributes still needed? AFAIK only DLNA profiles are stored in XML.

Comment (barronpm) [reply MediaBrowser.Model/Dlna/DeviceProfile.cs:26]: My plan was to move this code to a class called `DlnaDeviceProfile` that extends this class, so some of them are still needed unfortunately AFAIK. If there's an easy way to apply the attributes from the dlna plugin I'd prefer that, but idk.

Comment (nielsvanvelzen) [reply MediaBrowser.Model/Dlna/DeviceProfile.cs:26]: Might be possible with an override.

**Review by nielsvanvelzen — APPROVED** [review]

**Review by Shadowghost — APPROVED** [review]

File: MediaBrowser.Model/Dlna/DeviceProfile.cs (hunk 1 above)
Comment (barronpm) [reply MediaBrowser.Model/Dlna/DeviceProfile.cs:26]: Looks like it's possible, but there are some caveats associated, so I'll leave this as-is for now and play around with it later

**Review by Bond-009 — APPROVED** [review]

**Review by crobibero — APPROVED** [review]

**Pull request merged**